sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/CHANGELOG.md

```diff
@@ -1,14 +1,13 @@
 # Release History
 
-## 1.16.0-beta.1 (Unreleased)
+## 1.16.0 (2022-10-13)
```

Comment:
Shouldn't we bump the version to 1.15.2 ?

Reply:
The underlying dependency has a minor bump... revapi was unhappy about it

Reply:
What dependency was this?